webapp/src/modules/mapBuilder.js

```diff
@@ -214,28 +215,34 @@ export default class MapBuilder {
         if (!loc) {
           loc = e.latlng;
           this._mcg.clearLayers();
-          this._map.setView([e.latitude + this._latOffset, e.longitude], 20);
+          this._map.setView([e.latitude + this._latOffset, e.longitude], this._maxZoom);
           line = this._l.polyline([[e.latlng.lat, e.latlng.lng]]);
-          this._tripMarker = this._l
-            .animatedMarker(line.getLatLngs(), {
-              icon: this._scooterIcon,
-            })
-            .addTo(this._map);
+          this._tripMarker = this._l.animatedMarker(line.getLatLngs(), {
+            icon: this._scooterIcon,
+            autoStart: false,
```

Comment:
This prevents animation from the start. An issue was happening with animation. Probably due to `line` only having one latLng location.